up

```diff
@@ -1,56 +1,9 @@
-#!/bin/sh
-
+#!/bin/bash
 set -eu
 
-: "${DATALAD_REGISTRY_INSTANCE_PATH:=$PWD/instance}"
-export DATALAD_REGISTRY_INSTANCE_PATH
-: "${DATALAD_REGISTRY_DATASET_CACHE:=$DATALAD_REGISTRY_INSTANCE_PATH/cache}"
-export DATALAD_REGISTRY_DATASET_CACHE
-: "${DATALAD_REGISTRY_LOG_LEVEL:=DEBUG}"
-export DATALAD_REGISTRY_LOG_LEVEL
-export FLASK_APP=datalad_registry.factory:create_app
-export FLASK_ENV=development
-
-mkdir -p "$DATALAD_REGISTRY_INSTANCE_PATH"
-python -m flask init-db
-docker-compose -f docker-compose.broker.yml pull
-docker-compose -f docker-compose.broker.yml up &
-
-broker_ok=
-for i in $(seq 30)
-do
-    echo "Checking for broker connection (try $i)..."
-    if ! nc -nvz 127.0.0.1 5672
-    then
-        sleep 1
-    else
-        broker_ok=1
-        break
-    fi
-done
-test -z "$broker_ok" && exit 1
-
-celery () {
-    python -m celery -A datalad_registry.runcelery.celery "$@"
-}
-
-celery worker --loglevel "$DATALAD_REGISTRY_LOG_LEVEL" &
-celery beat --loglevel "$DATALAD_REGISTRY_LOG_LEVEL" \
-       --schedule "$DATALAD_REGISTRY_INSTANCE_PATH"/celerybeat-schedule &
-
-celery_ok=
-for i in $(seq 10)
-do
-    echo "Checking celery status (try $i)..."
-    if celery status --timeout 1 --json | grep -q pong
-    then
-        echo "Celery node found"
-        celery_ok=1
-        break
-    else
-        sleep 5
-    fi
-done
-test -z "$celery_ok" && exit 1
-
-python -m flask run
+mkdir -p "${DATALAD_REGISTRY_INSTANCE_PATH:-instance}"
+mkdir -p "${DATALAD_REGISTRY_DB_INSTANCE_PATH:-instance-db}"
+export UID_GID="$(id -u):$(id -g)"
+docker-compose pull
+docker-compose build
+docker-compose up -d
```

Comment:
The `.env` file is for Docker Compose, not Flask.  I really think that (for everything short of production) we should prefer running Flask via Docker Compose instead of directly.

Reply:
> we should prefer running Flask via Docker Compose instead of directly.

just to make sure -- there should be a "development mode" where code itself is bound/accessed from outside of the docker container so change to code triggers reload of flask app, right?

Reply:
@yarikoptic I personally don't see a need for that.

Reply:
how would we develop the code then??

Reply:
@yarikoptic Rebuild the image after ever code change?

Reply:
that is very inefficient IMHO, can't in development mode code be bound mount inside the container and ran from the container without changing the container?  (isn't that what dandi-api does?)